src/Tooltip/Tooltip.tsx

```diff
@@ -91,7 +93,13 @@ const Tooltip = ({
                 </div>
               )}
             </div>
-            <div role="content">{content}</div>
+            <div
+              role="content"
+              style={{ fontSize: "14px" }}
+              className={`${className}`}
```

Comment:
```suggestion
              className={className ?? ""}
```
No need for interpolation when it's the only variable being used inside.